lib/utils/Integers.ml

```diff
@@ -0,0 +1,207 @@
+let _ =
+  if Sys.int_size < 33 then
+    failwith "The size of the integers available is not large enough."
+
+(** First possible address : 0 *)
+let first_address = 0
+
+(** First possible address : 2^32 - 1 *)
+let last_address = (1 lsl 32) - 1
+
+let is_uint16 =
+  let two_16 = 1 lsl 16 in
+  fun i -> 0 <= i && i < two_16
```

Comment:
On devrait pouvoir écrire `let is_uint16 x = 0 <= x && x < (1 lsl 16)` tout simplement non ? La constante sera calculé une seule fois à la compilation. Je trouve ça un peu overkill d'écrire de cette façon.
Ton avis ?

Reply:
Oui oui, **à modifier**.

Reply:
Fixed in ecf8949a2acd1b3f3c7509bc4fcc6ff07f12e557.